### PR TITLE
ImagePicker Atom (merge ImageList first)

### PR DESCRIPTION
**Pull request by chamod-gamage**
Check out and review #114 first before you look at this one!
**WORK DONE**
+ Created ImagePicker.tsx (built off #114) with the necessary states; uses the following props
image

**_Implementation_**
image
image
image

**_Design_**
image
image
image

### CODE REVIEW

File: client/src/style/_config.scss [hunk 1]
```diff
@@ -20,6 +20,9 @@ $faded-modal-text-color: #7d7d7d;
 $normal-text-color: black;
 $white-text-color: white;
 $section-header-text-color: #bbbbbb;
+$tooltip-background-color: #efefef;
```
Comment (meganniu) [inline client/src/style/_config.scss:23]: Is it possible to keep the `tooltip` colours in `_config.scss` and delete `Tooltip.scss` and `Tooltip.tsx`?

File: client/src/components/atoms/ImagePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  selected: string;
+  isErroneous: boolean;
+}
+
+const ImagePicker: FunctionComponent<Props> = (props: Props) => {
+  const { selected, images, onImageChange, isErroneous } = props;
+  return (
+    <div className="imagepicker">
+      <Row className="toprow">
+        <div className="imagepicker-title">Image</div>
```
Comment (meganniu) [inline client/src/components/atoms/ImagePicker.tsx]: Is it possible to remove the title "Image" and the error text message "Please select an image"?
We're planning to wrap all input components (ImageList, TextField, etc.) in [FormItem](https://www.notion.so/uwblueprintexecs/34a44be943164278bfbf43de35e4ef7b?v=448f1b24885146a7b264dbf09eb60be1) which will add the header and error message.

Sorry, I should've been more clear in ur ticket description!

**Review by meganniu — APPROVED** [review]
Again, amazing work!! Thanks @chamod-gamage

File: client/src/style/_config.scss (hunk 1 above)
Comment (meganniu) [reply client/src/style/_config.scss:23]: These diffs might disappear if you rebase `ImageList` on `main` because ToolTip was merged to `main`

**Comment by bonnie-chin** [comment]
WAOWAOWAOOO SUPER IMPRESSED the work that you guys do is always 🤩🤩🤩

Just a few commentsss:
- can the error text be in line with the "Image" text? it seems to be sitting above
- maybe reducing the margin/padding between the error icon and the image display box but no biggie honestly

File: client/src/style/_config.scss (hunk 1 above)
Comment (chamod-gamage) [reply client/src/style/_config.scss:23]: Lol i was so confused how those appeared I didnt realize that you rebased 😆

**Comment by chamod-gamage** [comment]
> WAOWAOWAOOO SUPER IMPRESSED the work that you guys do is always 🤩🤩🤩
> 
> Just a few commentsss:
> 
> * can the error text be in line with the "Image" text? it seems to be sitting above
> * maybe reducing the margin/padding between the error icon and the image display box but no biggie honestly

LOL thank you - also it turns out the error messaging, alert icon, and the title Image is supposed to be part of another component that wraps so ig we're good to go on the design front for this component @bonnie-chin

**Review by jlight99 — APPROVED** [review]
lgtm!

**Pull request merged**